mobile/src/validation-schemas/chat/chat-message-create.validation-schema.ts

```diff
@@ -0,0 +1,20 @@
+import { getNameOf } from 'guruhub-shared/helpers/helpers';
+import Joi from 'joi';
+
+import {
+  ChatValidationMessage,
+  ChatValidationRule,
+} from '~/common/enums/enums';
+import { ChatMessageCreateRequestBodyDto } from '~/common/types/types';
+
+const chatMessageCreate = Joi.object({
+  [getNameOf<ChatMessageCreateRequestBodyDto>('message')]: Joi.string()
+    .trim()
+    .min(ChatValidationRule.MESSAGE_MIN_LENGTH)
+    .required()
+    .messages({
+      'string.min': ChatValidationMessage.MESSAGE_MIN_LENGTH,
+    }),
```

Comment:
Duplicate of `ChatValidationRule.MESSAGE_MIN_LENGTH`, leave for .min()

Reply:
There is no dublication: ChatValidation**Rule** and ChatValidation**Message**

Reply:
My bad